### PR TITLE
Test stream

### PR DESCRIPTION
**Pull request by mdellweg**
Also export ArtifactResonse in the plugin api.

### CODE REVIEW

File: CHANGES/plugin_api/2537.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Added ``pulpcore.plugin.content.ArtifactResponse`` to plugin API.
```
Comment (ipanova) [inline CHANGES/plugin_api/2537.feature:1]: for information purpose, can you add a line about what this response is about?

**Review by ipanova — APPROVED** [review]

File: template_config.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@ flake8: true
 github_org: pulp
 issue_tracker: github
 keep_ci_update_for_latest_branches: 5
-keep_ci_update_since_branch: "3.14"
+keep_ci_update_since_branch: '3.14'
```
Comment (bmbouter) [inline template_config.yml:37]: With black aren't all string with `"`?

Comment (bmbouter) [reply template_config.yml:37]: I guess this is yml not python....

**Review by bmbouter — APPROVED** [review]
Thank you!

File: template_config.yml (hunk 2 above)
Comment (mdellweg) [reply template_config.yml:37]: I didn't touch this line by hand.

**Pull request merged**